utilities_common/cli.py

```diff
@@ -183,8 +184,11 @@ def alias_to_name(self, interface_alias):
         # interface_alias not in port_dict. Just return interface_alias
         return interface_alias if sub_intf_sep_idx == -1 else interface_alias + VLAN_SUB_INTERFACE_SEPARATOR + vlan_id
 
-# Global class instance for SONiC interface name to alias conversion
-iface_alias_converter = InterfaceAliasConverter()
+def GetInterfaceAliasConverter():
```

Comment:
>GetInterfaceAliasConverter [](http://example.com/codeflow?start=4&length=26)

`Get` is not proper verb for your intention. If user keep `Get`, it actually like creating many converters.

For this common usage, suggest use lambda instead of a new function.

Reply:
Fixed, use lambda and remove this method.